spec/sidekiq_publisher/job_spec.rb

```diff
@@ -15,12 +15,12 @@
 
   describe ".create!" do
     it "sets a job_id if unset" do
-      job = described_class.create!(job_class: "Foo", args: [])
+      job = described_class.create!(job_class: "Foo", args: {})
```

Comment:
for some reason this spec and the one below were causing [this validation](https://github.com/ezcater/sidekiq_publisher/blob/ce0f1cc352d5e2b1411812d0f97e7667d027cbfc/lib/sidekiq_publisher/job.rb#L15) to fail for rails 6.1 appraisals. imo the error message is quite deceptive:
```
ActiveRecord::RecordInvalid:
  Validation failed: Args is reserved
```
i didn't investigate further but i do believe passing a hash is more appropriate for this spec

Reply:
I _think_ this is a quirk of validating inclusion/exclusion on a field that can accept an array. It looks like it takes the passed in array and iterates over it to validate each value, but because it's using `.all?` it will automatically return true on an empty array 

https://github.com/rails/rails/blob/d612542336d9a61381311c95a27d801bb4094779/activemodel/lib/active_model/validations/clusivity.rb#L27-L28

Reply:
gah i think my `grep` was too aggressive or incorrect or something because now i'm seeing the messages and everything lol. either way, thanks for investigating!